src/TodoManager/TodoResume/TodoResume.js

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
-import { useSelector } from 'react-redux';
+import { useSelector } from '../../common/hooks/useSelector';
 
 export function TodoResume() {
-  const todos = useSelector(state => state.todos);
+  const todos = useSelector();
```

Comment:
Algo raro que el api de use selector cambie aquí, yo mantendría la función selectora, pero bueno para efectos de lo que se quiere it's ok

Reply:
Sí, en realidad al tener un solo reducer y los Todos lo único en el state no cambiaba demasiado. En cualquier otro caso, cuando existe varios partes de estado, sería necesario el selector.